### DISCUSSION TITLE
[Feature] Allow more flexibility when trying to upload already uploaded assets

### DISCUSSION THREAD

**Discussion by martibc16** [immich-app/immich#4726] [Feature Request]
### The feature

At the moment, when trying to manually backup assets from the mobile apps the system is very strict with what you can upload.

Specifically, if I select multiple assets and one of them is already uploaded then the app doesn't give the option to upload any of them.
For those people who don't use automatic backup it's a pain when a photo amongst hunderds has been already uploaded (for some reason) and you have to find it and unselect it to upload the rest.

So, when selecting multiple assets the app should ignore the selected assets that have already been uploaded and should upload the ones that are not on the server.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This hasn't explicitly been implemented, but _a lot_ has changed since it was requested and I would expect it to work today.
